tools/tidb-binlog-kafka.md

```diff
@@ -93,6 +93,20 @@ cd tidb-binlog-latest-linux-amd64
     dir = "/path/pb-dir"
     ```
 
+- The drainer outputs `kafka` and you need to set the following parameters in the configuration file:
+
+    ```
+    [syncer]
+    db-type = "kafka"
+
+    # when db-type is kafka, you can uncomment this to config the down stream kafka, or it will be the same kafka addrs where drainer pull binlog from.
```

Comment:
Sometimes the original comment needs to be updated in grammar. Please pay attention to that as well.

- pull -> pulls